frontend/src/features/apartment/ApartmentMaxPricePage.tsx

```diff
@@ -188,8 +189,7 @@ const LoadedApartmentMaxPrice = ({apartment}: {apartment: IApartmentDetails}): J
                         <FormInputField
                             inputType="date"
                             label="Yhtiölainaosuuden päivämäärä"
-                            fieldPath="housing_company_loans_date"
-                            required
+                            fieldPath="apartment_share_of_housing_company_loans_date"
```

Comment:
how easy it is to prevent selecting dates from the future? This would eliminate the `Field has to be less than or equal to current date.` error :thinking:

Reply:
Pretty eaasy, added a this :)